tests/fixtures.py

```diff
@@ -175,6 +175,8 @@ def _running_eventlet_server(app):
     port = socket.getsockname()[1]
     url = f"{scheme}://{host}:{port}"
     app.config['URL'] = url
+    if "fork" not in multiprocessing.get_all_start_methods():
+        pytest.skip("requires the multiprocessing start_method 'fork', which is unavailable on this system")
```

Comment:
pytest.mark may have influence to the tests sorted to the different processes. It is used in the test collection phase. it never runs a marked test.

pytest.skip is used in the call phase, it stops a running tests.

Maybe we can use the allow_module_level flag here?
https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-skip

Reply:
that was more like a remark, I didn't request a change.

Reply:
> mark.skip was in the past prefered used intestead .skip.

I agree, as long as pytest.mark.skip can be used. But, to my knowledge, it is impossible to skip all tests that depend on a fixture with just pytest.mark.skip.